phases/phase-2/challenges/add-reaction.md

```diff
@@ -0,0 +1,105 @@
+# Add a reaction to post
+:::tip Note  
+This challenge allows you to receive **30 Desmos Tokens** upon completion.  
+
+Please note that in order to prevent spam, you will receive the reward just **for the first post** you react to. No tokens will be awarded for later posts.  
```

Comment:
Please note that in order to prevent spam, you will receive the reward just **for the first post** you react to. No tokens will be awarded for later posts. 
-> 
You will only receive reward **for the first reaction** you have made. No reward for later reactions you have made.